sites/diverseeducation.com/server/templates/static-pages/mppwsa-yearly.marko

```diff
@@ -37,6 +42,18 @@ $ const ideas = currentYearData ? currentYearData.ideas: title;
         <div class="row">
           <div class="col">
             <h1 class="page-wrapper__website-section-name">$!{title}</h1>
+            <div class="row">
+              <div class="col" align="center">
+                <marko-web-img
+                  class=["lazyload"]
+                  alt="logo"
+                  src=currentYearData.mpplogoSrc
+                  srcset=`${currentYearData.mpplogoSrc} x2`
+                  lazyload=true
+                />
+              </div>
+            </div>
+            <br/>
```

Comment:
This makes the assumption an image exists this should likely follow the same checking as the tables do `<if(currentYearData && get(currentYearData, 'mppLogoSrc')>`

Reply:
@Shinsina 
<img width="872" alt="Screen Shot 2023-03-16 at 2 51 10 PM" src="https://user-images.githubusercontent.com/64623209/225737094-6f26ca6c-5a85-44fd-8aaf-0913954af07b.png">